autogpt_platform/frontend/src/lib/autogpt-server-api/types.ts

```diff
@@ -188,17 +188,19 @@ export type LinkCreatable = Omit<Link, "id" | "is_static"> & {
   id?: string;
 };
 
-/* Mirror of autogpt_server/data/graph.py:ExecutionMeta */
-export type ExecutionMeta = {
+/* Mirror of backend/data/graph.py:GraphExecution */
+export type GraphExecution = {
   execution_id: string;
   started_at: number;
   ended_at: number;
```

Comment:
`started_at` and `ended_at` are not unix timestamps but ISO-formatted timestamps!

Reply:
This turns out to have broken the `FlowRunsTimeline`, maybe more